### ISSUE TITLE
Backend frontname from app/etc/env.php is not taken into account

### ISSUE THREAD

**Issue by ajpevers** [fastly/fastly-magento2#131]
The following VCL entries are now static.
Request to read them from `app/etc/env.php` and take custom admin urls into account:

```
    # Increase first byte timeouts for /admin* URLs to 3 minutes
    if ( req.url ~ "^/(index\.php/)?admin(_.*)?/" ) {
      set bereq.first_byte_timeout = 180s;
    }
```

```
    # Pass all admin actions
    } else if ( req.url ~ "^/(index\.php/)?admin(_.*)?/" ) {
        set req.http.x-pass = "1";
    } else {
```

**Comment by vvuksan** [comment]
We'll look into it.

**Comment by vvuksan** [comment]
This has been resolved with the pull request #132
